build.gradle

```diff
@@ -138,10 +155,8 @@ configurations.all {
 
 dependencies {
 
-    implementation(group: 'com.github.LBNL-UCB-STI', name: 'beam-utilities', version: 'v0.2.19') {
-        exclude group: 'com.github.LBNL-UCB-STI', module: 'r5'
-        exclude group: 'org.matsim', module: 'matsim'
-    }
+    // TODO FIXME
+    implementation(group: 'com.github.LBNL-UCB-STI', name: 'beam-utilities', version: '0.3.4')
```

Comment:
These changes are based off of the master branch - https://github.com/LBNL-UCB-STI/beam-utilities